src/test/resources/application-test.yml

```diff
@@ -66,7 +66,7 @@ hmpps.sqs:
       queueName: "adjudication-${random.uuid}"
       dlqName: "adjudication-dlq-${random.uuid}"
       subscribeTopicId: domainevents
-      subscribeFilter: "{\"eventType\":[\"adjudication.report.created\", \"adjudication.damages.updated\", \"adjudication.evidence.updated\", \"adjudication.hearing.created\", \"adjudication.hearing.updated\", \"adjudication.hearing.deleted\", \"adjudication.hearingCompleted.created\", \"adjudication.hearingCompleted.deleted\", \"adjudication.hearingReferral.created\", \"adjudication.hearingReferral.deleted\", \"adjudication.hearingAdjourn.created\", \"adjudication.hearingAdjourn.deleted\", \"adjudication.punishments.created\", \"adjudication.punishments.updated\", \"adjudication.referral.outcome.prosecution\", \"adjudication.referral.outcome.notProceed\", \"adjudication.referral.outcome.referGov\", \"adjudication.referral.outcome.deleted\", \"adjudication.outcome.referPolice\", \"adjudication.outcome.notProceed\", \"adjudication.referral.deleted\", \"adjudication.referral.outcome.deleted\", \"adjudication.outcome.notProceed.deleted\"]}"
+      subscribeFilter: "{\"eventType\":[\"adjudication.report.created\", \"adjudication.damages.updated\", \"adjudication.evidence.updated\", \"adjudication.hearing.created\", \"adjudication.hearing.updated\", \"adjudication.hearing.deleted\", \"adjudication.hearingCompleted.created\", \"adjudication.hearingCompleted.deleted\", \"adjudication.hearingReferral.created\", \"adjudication.hearingReferral.deleted\", \"adjudication.hearingAdjourn.created\", \"adjudication.hearingAdjourn.deleted\", \"adjudication.punishments.created\", \"adjudication.punishments.updated\", \"adjudication.referral.outcome.prosecution\", \"adjudication.referral.outcome.notProceed\", \"adjudication.referral.outcome.referGov\", \"adjudication.referral.outcome.deleted\", \"adjudication.outcome.referPolice\", \"adjudication.outcome.notProceed\", \"adjudication.referral.deleted\", \"adjudication.referral.outcome.deleted\", \"adjudication.outcome.notProceed.deleted\", \"adjudication.hearingOutcome.updated\"]}"
```

Comment:
Steve mentioned this; I wonder if this can be formatted better, not sure how given it is yaml